content/blog/choose-tidymodels-adventure/index.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+output: hugodown::hugo_document
+slug: choose-tidymodels-adventure
+title: Choose your own tidymodels adventure
+date: 2021-05-24
+author: Julia Silge
+description: >
+    The tidymodels ecosystem is modular and flexible, which can sometimes make
+    choosing an approach overwhelming for newcomers. This post offers opinionated 
+    guidance on where to start!
+
+photo:
+  url: https://unsplash.com/photos/0LXFLzDOfuA
+  author: Jonathan Fox
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [learn] 
+tags: [tidymodels, workflowsets, workflows, parsnip]
+---
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. You can install the tidymodels suite of packages from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("tidymodels")
+```
+
+The packages in tidymodels are designed to work together in a unified ecosystem, but they are flexible and modular; you can use tidymodels packages for certain parts of a modeling analysis without committing to it entirely, when appropriate.
+
+```{r setup}
+library(tidymodels)
+```
+
+We purposefully write code in small, modular packages both to make them easier to maintain and easier to use in production systems. This does mean that it can be challenging for a newcomer to know where their specific problem fits into this ecosystem. We generally recommend a couple of resources for folks just getting started:
```

Comment:
"where their specific problem fits into" sounds awkward (should it be in instead of into?) but I'm bad with prepositions so I'm not sure